doc/source/all-styles.rst

```diff
@@ -0,0 +1,46 @@
+
+Style
+=====
+
+In the PyAnsys ecosystem, three key styles — Packaging, Coding, and Documentation 
+serve as foundational pillars to enhance the overall developer experience.
+
+The Packaging style focuses on creating clear, open-source APIs hosted on GitHub,
```

Comment:
```suggestion
Packaging style focuses on creating clear, open source APIs hosted on GitHub,
```
Google dev guide says it is "open source" even when used as a compound adjective.